packages/ui/src/theme/css/component/sliderField.scss

```diff
@@ -0,0 +1,83 @@
+.amplify-sliderfield {
+  flex-direction: column;
+}
+
+.amplify-sliderfield__label {
+  display: flex;
+  justify-content: space-between;
+}
+
+.amplify-sliderfield__group {
+  gap: var(--amplify-components-sliderfield-group-gap);
+}
+
+.amplify-sliderfield__root {
+  position: relative;
+  display: flex;
+  align-items: center;
+  user-select: none;
+  touch-action: none;
+
+  &[data-orientation='horizontal'] {
+    height: var(--amplify-components-sliderfield-root-horizontal-height);
+  }
+
+  &[data-orientation='vertical'] {
+    flex-direction: column;
+    width: var(--amplify-components-sliderfield-root-horizontal-width);
+  }
+}
+
+.amplify-sliderfield__track {
+  position: relative;
+  flex-grow: 1;
+  border-radius: 9999px;
```

Comment:
Should this value come from the token files?

Reply:
Well, I was thinking this was a special value. But it does make sense to make it as a token, since users may don't want the track to be rounded